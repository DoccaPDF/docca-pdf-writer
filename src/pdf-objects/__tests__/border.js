import Border from '../border';

const border = Border({
  W: 2, S: 'S'
});

describe('pdf-objects', () => {
  describe('border', () => {
    it('creates a new object', () => {
      expect(border).toMatchSnapshot();
    });

    it('flattens keys values', () => {
      expect(border.flattenKeys()).toMatchSnapshot();
    });

    it('returns a PDF object', () => {
      expect(border.toPDF()).toMatchSnapshot();
    });
  });
});
