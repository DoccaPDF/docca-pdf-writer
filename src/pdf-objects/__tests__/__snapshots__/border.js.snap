// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pdf-objects border creates a new object 1`] = `
Object {
  "S": "S",
  "W": 2,
}
`;

exports[`pdf-objects border flattens keys values 1`] = `
Object {
  "D": undefined,
  "S": "S",
  "Type": "/Border",
  "W": 2,
}
`;

exports[`pdf-objects border returns a PDF object 1`] = `
"undefined 0 obj
<<
/Type /Border
/W 2
/S S
>>
endobj
"
`;
